### DISCUSSION TITLE
Preguntas iniciales

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [workflix/workflix-full#3] [General]
Con respecto a si son ambos proyectos Web y movil, ambos son de la misma temática, tienen que estar al día de hoy funcionales, en el caso de tu equipo, luego de un año de la aplicación web y también de la móvil, van algunas preguntas y supuestos 

Relevamiento de lo que ya tienen desarrollado.
¿Sigue funcional? 
¿Se carga en el servidor?
¿Tiene vinculación con la móvil de alguna manera? o desde la móvil alguna pantalla para que pueda visitar la web?
¿Qué funcionalidad como mejora superadora podrían agregar en ambas aplicaciones?
Revisar accesibilidad
Realizar pruebas de funcionamiento (testing) sobre todo en web que no se aplicaron

**Comment by BraianTroncoso** [comment]
¿Sigue funcional? Si, sigue funcional
¿Se carga en el servidor? todo correctamente
¿Tiene vinculación con la móvil de alguna manera? o desde la móvil alguna pantalla para que pueda visitar la web? No, eso vamos a implementar
¿Qué funcionalidad como mejora superadora podrían agregar en ambas aplicaciones? Vinculación total, lading, también estilos, se charlara en las meets.
Revisar accesibilidad: Todo ok.
Realizar pruebas de funcionamiento (testing) sobre todo en web que no se aplicaron: Pude levantar le back y front, funcionaba bien.

**Comment by yan0106** [comment]
Intenté levantar el backend de la app web y no lo logré, me sale error con makemigration. Consulto con mis compañeros para encontrar el error.

**Comment by MarielaGomez** [comment]
Se realizó revisión de documentación y completar según sprint la wiki correspondiente.

**Comment by NataliaAlvarezIspc** [comment]
### Pasando en limpio las respuestas a las preguntas iniciales

**¿Sigue funcional?**  
Si, siguen funcionales, tanto la página web como la aplicación móvil  

**¿Se carga en el servidor?**  
Ambas aplicaciones se cargan en local  

**¿Tiene vinculación con la móvil de alguna manera, o desde la móvil alguna pantalla para que pueda visitar la web?**
No, esa es una de las mejoras que vamos a implementar

**¿Qué funcionalidad como mejora superadora podrían agregar en ambas aplicaciones?** 
Vinculación total, congruencia de estilos, agregaremos una página institucional, con información de Workflix, que direccione a la web o a descargar la app.
Haremos coincidir la misma base de datos de profesionales tanto en la web como en la aplicación móvil.
Incluiremos, en la aplicación móvil, un botón que direccione a la página web.
Agregaremos una activity para poder visualizar los profesionales mejor valorados.
Pondremos en funcionamiento la sección para restablecer la contraseña.
Activaremos la búsqueda por rubro profesional en ambas aplicaciones.
En la aplicación móvil, se podrá contactar por WhatsApp a los profesionales.
Realizaremos pruebas de testing, sobre todo en el sitio web, en donde no se llevaron adelante con anterioridad.
Revisar accesibilidad: Todo ok.

**Realizar pruebas de funcionamiento (testing) sobre todo en web que no se aplicaron:** 
Realizaremos pruebas de testing, sobre todo en la página web, en donde no se han realizado
